hazelcast/cluster.py

```diff
@@ -122,11 +137,13 @@ def start(self, connection_manager, membership_listeners):
             self.add_listener(*listener)
 
     def get_member(self, member_uuid):
+        # type: (uuid.UUID) -> List[_MemberListSnapshot]
```

Comment:
The return type is not correct, it should have been a `MemberInfo`

Also, I see that `start` method above this has missing type hints